src/status_im/ui/screens/profile/group_chat/views.cljs

```diff
@@ -106,6 +106,7 @@
            :extended-header   (profile-header/extended-header
                                {:title         chat-name
                                 :color         color
+                                :monospace     false
```

Comment:
is it monospace by default?

Reply:
I've made it by default in this PR, but I will inverse it now. I've seen that local contact uses the subtitle and it does not make sens to have it now as monospace by default